actions/src/robocorp/actions/__init__.py

```diff
@@ -8,7 +9,7 @@
 version_info = [int(x) for x in __version__.split(".")]
 
 
-def action(func):
+def action(*args, **kwargs):
```

Comment:
For the `action` (not `task`), we should add the `@overload`s with the signatures which are valid (see: https://github.com/robocorp/robo/blob/master/windows/src/robocorp/windows/_window_element.py#L136 as an example).